web/src/screens/settings/Irc.tsx

```diff
@@ -169,23 +169,23 @@ const IrcSettings = () => {
       </div>
 
       {data && data.length > 0 ? (
-        <ul className="mt-6 min-w-full relative">
+        <ul className="mt-6 min-w-full relative text-sm">
           <li className="grid grid-cols-12 gap-4 border-b border-gray-200 dark:border-gray-700">
-            <div className="flex col-span-2 md:col-span-1 pl-0 sm:px-3 py-3 text-left text-xs font-medium text-gray-500 dark:text-gray-400 uppercase tracking-wider cursor-pointer"
+            <div className="flex col-span-2 md:col-span-1 pl-2 sm:px-3 py-3 text-left text-xs font-medium text-gray-500 dark:text-gray-400 uppercase tracking-wider cursor-pointer"
               onClick={() => sortedNetworks.requestSort("enabled")}>
-              Enabled <span className="sort-indicator">{sortedNetworks.getSortIndicator("enabled")}</span>
+              Enabled <span className="sort-indicator pl-1">{sortedNetworks.getSortIndicator("enabled")}</span>
             </div>
-            <div className="col-span-10 md:col-span-3 px-8 py-3 text-left text-xs font-medium text-gray-500 dark:text-gray-400 uppercase tracking-wider cursor-pointer"
+            <div className="col-span-10 md:col-span-3 px-11 py-3 text-left text-xs font-medium text-gray-500 dark:text-gray-400 uppercase tracking-wider cursor-pointer"
               onClick={() => sortedNetworks.requestSort("name")}>
-              Network <span className="sort-indicator">{sortedNetworks.getSortIndicator("name")}</span>
+              Network <span className="sort-indicator pl-1">{sortedNetworks.getSortIndicator("name")}</span>
             </div>
             <div className="hidden md:flex col-span-4 px-6 py-3 text-left text-xs font-medium text-gray-500 dark:text-gray-400 uppercase tracking-wider cursor-pointer"
               onClick={() => sortedNetworks.requestSort("server")}>
-              Server <span className="sort-indicator">{sortedNetworks.getSortIndicator("server")}</span>
+              Server <span className="sort-indicator pl-1">{sortedNetworks.getSortIndicator("server")}</span>
             </div>
-            <div className="hidden md:flex col-span-3 px-5 lg:px-6 py-3 text-left text-xs font-medium text-gray-500 dark:text-gray-400 uppercase tracking-wider cursor-pointer"
+            <div className="hidden md:flex col-span-3 px-5 py-3 text-left text-xs font-medium text-gray-500 dark:text-gray-400 uppercase tracking-wider cursor-pointer"
```

Comment:
I just tested it and there were no changes after i consolidated the tailwind classes to the parent container so that's correct!
Thanks for pointing that out! 😄